CHANGELOG.rst

```diff
@@ -2,10 +2,20 @@
 History
 =======
 
-0.16.0 (unreleased)
+0.16.0 (2023-02-09)
 ===================
 
-*
+* Add support to export and import configuration data from plugins.
```

Comment:
Just to comment, usually changelogs use the past tense ("Added" instead of "Add" for example). I realized this is not consistent in previous releases, so feel free to leave as is.

Reply:
Ah, ok.
Let's use the past tense here and for new items.

- [ ] new changelog entries in past tense;